swagger/enterprise-developer-portal-swagger.yaml

```diff
@@ -8440,7 +8735,7 @@ components:
         AuthTokenType:
           type: string
           enum:
-            - AUTH_TOKEN_CUSTOM
+            - AUTH_TOKEN_TYPE_CUSTOM
```

Comment:
@tbuchaillot do we have this change?

Reply:
This may be introduced during repo migration. But `AUTH_TOKEN_TYPE_CUSTOM` is the correct option.

Reference:
- https://tyktech.slack.com/archives/C08C4LFM336/p1758867862479279
- https://github.com/TykTechnologies/tyk-docs-hugo/pull/6997
- https://github.com/TykTechnologies/portal/pull/1716